Misc/NEWS.d/next/Library/2018-07-13-13-42-10.bpo-34097.F5Dk5o.rst

```diff
@@ -0,0 +1,2 @@
+ZipFile can zip files older than 1980-01-01 at the cost of mangling the
```

Comment:
You should mention the new timestamps parameter. And maybe explain "mangling": dates before 1980-01-01 are replaced by 1980-01-01 at 00:00.